uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1000.php

```diff
@@ -36,11 +36,20 @@ private function up1000_updateHalfDayPmLabel() {
    * @param string $label
    */
   private function up1000_updateOptionValueLabel($name, $label) {
-    civicrm_api3('OptionValue', 'get', [
-      'sequential' => 1,
-      'option_group_id' => 'hrleaveandabsences_leave_request_day_type',
-      'name' => ['IN' => [$name]],
-      'api.OptionValue.create' => ['id' => '$value.id', 'label' => $label],
-    ]);
+    try {
+      civicrm_api3('OptionValue', 'get', [
```

Comment:
Well I'm not so comfortable with signing off on code that I don't understand. That said I don't want to hold up L&A development for this. It would be nice if there was an explanation, but if it's going to take too much time then I can approve @tunbola's PR.